include/fem-dem/parameters_cfd_dem.h

```diff
@@ -72,6 +78,8 @@ namespace Parameters
     unsigned int                   particle_refinement_factor;
     double                         qcm_sphere_diameter;
     bool                           qcm_sphere_equal_cell_volume;
+    VoidFractionQCMRule            qcm_quadrature_rule;
```

Comment:
I think we should not make it specific to QCM even though it is in a way. Let's just apply it everywhere and call that VoidFractionQuadratureRule
We could eventually have a general parameter called quadraturerule if we change quadrature for the other physics. Maybe now is the time to start it cleanly.